File: Arjuna/versions/PR3.3_97_03_27/Tests/Basic/Local/UidUniqueness/UidUniqueness.cc
/*
 * Copyright (C) 1993
 * 
 * Department of Computing Science,
 * The University,
 * Newcastle upon Tyne,
 * UK.
 */

/*
 * $Id: UidUniqueness.cc,v 1.2 1994/09/14 08:56:22 ngdp Exp $
 */

#include <System/iostream.h>
#include <Common/Uid.h>

int main(int, char*[])
{
    Uid uids[16];

    for (int i = 0; i < 16; i++)
	cout << uids[i] << '\n' << flush;

    return 0;
}

File: JTSArjuna/JTSArjuna-2.0_Alpha_2_1/ArjunaCommon/tests/Uid/UidUniqueness.java
/*
 * Copyright (C) 1998, 1999, 2000,
 *
 * Arjuna Solutions Limited,
 * Newcastle upon Tyne,
 * Tyne and Wear,
 * UK.  
 *
 * $Id: UidUniqueness.java,v 1.1 2000/02/25 14:03:25 nmcl Exp $
 */

import com.arjuna.ArjunaCommon.Common.*;

public class UidUniqueness
{
    
public static void main (String[] args)
    {
	for (int i = 0; i < 100; i++)
	{
	    Uid u = new Uid();
	    System.out.println(u + " " + u.hashCode());
	}
    }
    
};
